src/components/Search/CompareOverTime.tsx

```diff
@@ -1,25 +1,106 @@
+import { RevisionsList, Repository } from '../../types/state';
+import CompareButton from './CompareButton';
+import FrameworkDropdown from './FrameworkDropdown';
+import SearchOverTime from './SearchOverTime';
 
 const strings = Strings.components.searchDefault;
+const stringsNew =
+  Strings.components.searchDefault.overTime.collapsed.revisions;
+
+interface CompareWithTimeProps {
+  isEditable: boolean;
+  baseRevs: RevisionsList[];
+  newRevs: RevisionsList[];
+  baseRepos: Repository['name'][];
+  newRepos: Repository['name'][];
+}
+
+function CompareOverTime({ isEditable }: CompareWithTimeProps) {
+  const [expanded, setExpanded] = useState(false);
 
-function CompareOverTime() {
   const mode = useAppSelector((state) => state.theme.mode);
   const styles = CompareCardsStyles(mode);
+  const dropDownStyles = SearchStyles(mode);
+  const search = useAppSelector((state) => state.search);
+  const searchResults = search.new.searchResults;
+  //temporary hash to hide the component until functionality is complete
+  const location = useLocation();
+  const hash = location.hash;
+
+  //temporary styles to hide the component based on hash
+  const containerStyles = {
+    container: style({
+      display: hash === '#comparetime' ? 'block' : 'none',
+    }),
+  };
+
+  const bottomStyles = {
+    container: style({
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'flex-end',
+    }),
+  };
+
+  const toggleIsExpanded = () => {
+    setExpanded(!expanded);
+  };
+
   return (
-    <div
-      className={`compare-card-container compare-card-container--time ${styles.container}`}
+    <Grid
+      className={`wrapper--overtime ${styles.wrapper} ${containerStyles.container}`}
     >
-      <div className={`compare-card-text ${styles.cardText}`}>
-        <div className='compare-card-title'>{strings.overTime.title}</div>
-        <div className='compare-card-tagline'>{strings.overTime.tagline}</div>
+      <div
+        className={`compare-card-container compare-card-container--${
+          expanded ? 'expanded' : 'hidden'
+        } ${styles.container}`}
+        onClick={toggleIsExpanded}
+        data-testid='time-state'
+      >
+        <div className={`compare-card-text ${styles.cardText}`}>
+          <div className='compare-card-title'>{strings.overTime.title}</div>
+          <div className='compare-card-tagline'>{strings.overTime.tagline}</div>
+        </div>
+        <div
+          className='compare-card-img compare-card-img--time'
+          aria-label='a clock'
+        />
       </div>
       <div
-        className='compare-card-img compare-card-img--time'
-        aria-label='a clock'
-      />
-    </div>
+        className={`compare-card-container content-base content-base--${
+          expanded ? 'expanded' : 'hidden'
+        } ${styles.container} `}
+      >
+        <Divider className='divider' />
+        <Form action='/compare-results' className='form-wrapper'>
```

Comment:
What do you think about adding an aria-label to the form, so that you can target it more easily in tests and improve accessibility?